rdflib/plugins/sparql/evaluate.py

```diff
@@ -299,7 +301,7 @@ def evalPart(ctx, part):
         raise Exception("I dont know: %s" % part.name)
 
 
-def evalServiceQuery(ctx, part):
+def evalServiceQuery(ctx: QueryContext, part):
```

Comment:
Any reason this `part` doesn't have `CompValue` like the others?

Reply:
No particular reason, I was just adding types which were either very clear or part of the code being executed, the untyped values remain `typing.Any`, but should ideally eventually also be typed with gradual typing.